src/core/api/public_api.ts

```diff
@@ -2416,7 +2487,7 @@ class Player extends EventEmitter<IPublicAPIEvent> {
     this._priv_lastContentPlaybackInfos.manifest = manifest;
 
     const { initialAudioTrack, initialTextTrack } = contentInfos;
-    this._priv_trackChoiceManager = new TrackChoiceManager();
+    this._priv_trackChoiceManager = new TrackChoiceManager(this._priv_trickModeEnabled$);
```

Comment:
Looks weird to add a BehaviorSubject in argument considering that we already call `setVideoTrickMode`/`unsetVideoTrickMode` so it already has all the information it needs.